### ISSUE TITLE
Emulator-start goal gives a timeoutException even if the emulator is been started

### ISSUE THREAD

**Issue by amanganiello90** [simpligility/android-maven-plugin#752]
The 4.4.3 version doesn't work on Windows for the wbs script. However the 4.3.0 version run but with the problem exposed in the issue title. That is the emulator is started (and up running) but maven fails with a timeout exception. 
Temporarily I have resolved the problem with a combination of maven antrun plugin and a node.js script.
You can download this on: https://github.com/amanganiello90/maven-android-emulator-plugin

**Comment by Shusshu** [comment]
Maybe you can fix the latest maven android plugin emulator mojo ? I don't think anybody else is going to have a go at it....

**Comment by amanganiello90** [comment]
What should I have to fix?? You can use my temporarily solution on my repository.
Or, if you have other requests, you can open an issue on my maven-android-emulator-plugin and give a donation in the https://github.com/amanganiello90/maven-gitflow-plugin because I already work in a full-time job daily. Thanks

**Comment by william-ferguson-au** [comment]
I suggest that we cut support for the emulator in the plugin. IMHO the
plugin already suffers badly from being a Swiss army knife.

**Comment by amanganiello90** [comment]
@william-ferguson-au  I don't understant what we will do. Fork the maven-android-plugin and resolve it or eliminate the goal that doesn't work?

**Comment by william-ferguson-au** [comment]
I vote for eliminating the goal.
Let another plugin focus on the emulator.

**Comment by amanganiello90** [comment]
Maybe I can modify the mojo goal class to insert my node script that I have created in my repo. Then I can use trireme (https://github.com/apigee/trireme) to call this scripts in the mojo.  So, after this, I open a pull request to integrate it.

**Comment by mosabua** [comment]
I would rather not use some nodejs code but I would like to continue having the goal. I think I am happy to just release the plugin as a new version to get the changes out and just have those goals broken for now. And then we can decide on removing or fixing based on the feedback.

**Comment by amanganiello90** [comment]
@mosabua I don't want to use nodejs code instead of the maven plugin. I'm working on a maven plugin that executes nodejs code in java. So It will be very easy to integrate my script using my core-class in the maven-android-plugin  mojo in order to fix the problem in the start-emulator. When it is ready the core, I will be able to create a pull request on your plugin. As your goal doesn't work and you want to break it, you have nothing to lose to integrate a new mojo that can assolve the start-emulator job.

**Comment by mosabua** [comment]
Sounds good. I would love to pull your stuff in.. however please do not submit it as a new mojo. Just fix the existing mojo ;-)

**Comment by amanganiello90** [comment]
Yes, I have said new mojo to refer the existing goal integrated and fixed

**Comment by amanganiello90** [comment]
I have opened a pull request that fix the goal entirely with java code.
Review it and maybe you can release the fix.
Thanks.
